packages/TorneloScoresheet/android/gradle.properties

```diff
@@ -26,3 +26,4 @@ android.enableJetifier=true
 
 # Version of flipper SDK to use with React Native
 FLIPPER_VERSION=0.99.0
+org.gradle.jvmargs=-Xmx1536M --add-exports=java.base/sun.nio.ch=ALL-UNNAMED --add-opens=java.base/java.lang=ALL-UNNAMED --add-opens=java.base/java.lang.reflect=ALL-UNNAMED --add-opens=java.base/java.io=ALL-UNNAMED --add-exports=jdk.unsupported/sun.misc=ALL-UNNAMED
```

Comment:
do you know what this is for?

Reply:
Llio and I were both experiencing the following error: 

https://stackoverflow.com/questions/67782975/how-to-fix-the-module-java-base-does-not-opens-java-io-to-unnamed-module 

This fix worked for both of us - if we don't push this we need to add this line of code manually every time we pull. Hoping we can keep it.